### DISCUSSION TITLE
MVP#2-QualifiedQuery-Gene(Protein)-to-Chemical

### DISCUSSION THREAD

**Discussion by karafecho** [NCATSTranslator/Clinical-Data-Committee-Tracking-Voting#18] [Ideas]
This discussion thread is intended to promote discussion among the TCDC regarding the MVP#2 templated query and approaches for incorporating clinical knowledge into inferred (creative) mode responses:

_What chemicals [qualified] affect (increases/decreases) a given protein/gene?
Conversely, what proteins/genes are [qualified] affected by (increased/decrease) by a given chemical entity?_

**Comment by karafecho** [comment]
One possibility, suggested by Andy, is for clinical KPs to contribute to MVP#2 by way of genes that are upregulated (or increased in activity or abundance) in disease X.

Matt suggested expressing ‘Gene X shows increased expression in Disease Y’ as follows:
```
subject: Gene X
subject_aspect: expression
subject_direction: increased
predicate: associated_with
object: Disease Y
```

I think there are several ways for the clinical and non-clinical KPs to contribute to a CARA workflow developed along these lines. For instance, I can imagine a workflow comprised of the following contributions or 'hops', some qualified, some not: MolePro provides qualified gene/protein-chemical assertions, Text Mining Provider / Genetics Provider provides gene/protein-disease assertions, and the clinical KPs provide disease-chemical assertions. I suspect there's a way to pull in disease-tissue-gene assertions from CHP, too.

I obviously need to think this through more, but I thought I'd move our discussion from today along a bit.

**Comment by karafecho** [comment]
**Thoughts from Sharat, Qian, and Christine, 01/09/2023:**

But there is one more Option that Christine brought up this morning for TCDC, so I will put it here.

Since our canonical Q’s, in relation to treating a disease, are:
_What [drugs] may <up/down>regulate a (specific gene)?
What [genes] may be <up/down>regulated by a (specific drug)?_

Therefore, when a researcher is looking for a drug to treat (eg) IPF, they will know that MUC5B is associated with IPF.  So, TCDC could call the right KP (genetics?) and ask what all diseases are associated with MUC5B, and [clinical KPs] return drugs used to treat that disease, along with the efficacy (if possible).

She said that TCDC could also look at what tissues associated with those diseases, and drugs that have an effect on those tissues.

By the way, when we say gene, we also imply associated protein in these queries.

Essentially, number six from discussion thread #21, then number two from discussion thread #21, but with one other step.

**Comment by marcdubybroad** [comment]
@karafecho , unclear if this would help, but the genetics KP has plans on adding qualifiers to our current gene/disease loss of function (LoF) data, and adding another 600k exome sequencing study that has the same data type; this should provide the direction of effect on a disease from the down regulation of a gene.
The data is computed by selecting only LoF coding variants (mostly rare) for genes, then aggregating their values at the gene level for each disease, getting  a p-value and direction of effect on the disease.
Timeline is getting this data to our dev instance in the next two weeks.

**Reply by karafecho** [reply]
Yes! The new data would definitely be responsive to the suggestions posted above. Thanks for sharing, Marc.
